src/ai_chunking.py

```
@@ -0,0 +1,222 @@
+            starts.append(message)
+            next_group = False
+
+        if False: # testing no validation
+            author_name = getattr(getattr(message, 'author', None), 'display_name', None)
+            content = getattr(message, 'content', None)
+
+            if not author_name:
+                logging.debug(f"Skipping incomplete message (no author): {message}")
+                continue  
+
+            if not not content:
```

Ruff (0.8.2)

Use `content` instead of `not (not content)`

Replace with `content`

(SIM208)